src/Inkluzitron/Modules/SendModule.cs

```diff
@@ -1,44 +1,70 @@
+using System.Linq;
 using System.Net.Http;
 using System.Threading.Tasks;
 using Discord;
 using Discord.Commands;
+using Inkluzitron.Models.Settings;
 
 namespace Inkluzitron.Modules
 {
     [Name("Odesílání zpráv")]
     public class SendModule : ModuleBase
     {
+        private SendSettings SendSettings { get; }
+
+        public SendModule(SendSettings sendSettings)
+        {
+            SendSettings = sendSettings;
+        }
+
         [Command("send")]
-        [Summary("Odešle anonymně zprávu (i včetně přílohy) do stejné roomky. Původní příkaz smaže.")]
-        public async Task SendAsync([Remainder][Name("zpráva")] string message = null)
+        [Summary("Odešle anonymně zprávu (včetně příloh) do zadané roomky. Funguje pouze v DM.")]
+        public async Task SendAsync([Name("název roomky")] string roomName, [Remainder][Name("zpráva")] string messageText = null)
```

Comment:
Místo string atributu bych tam dal přímo `ITextChannel channel`. Discord.NET to dokáže načíst a předat ti tam přímo instanci kanálu. Můžeš pak na channel odkazovat přes tag/id a myslím, že i název.